### ISSUE TITLE
MeshCtrl Shell connecting but not working

### ISSUE THREAD

**Issue by adsgreen** [Ylianst/MeshCentral#2143]
Just looking at the "shell" command but can't seem to get it to work as expected on windows 10:

It connects fine but then immediately exits and doesn't allow shell access.
```
C:\Program Files\Open Source\MeshCentral\node_modules\meshcentral>node meshctrl.js shell --id "fneK@...." --url wss://xxxx.com --loginuser xxxx --loginpass xxxx 
Connecting...
OK

C:\Program Files\Open Source\MeshCentral\node_modules\meshcentral>
```
MeshCentral V0.7.39

Other meshctrl commands working fine including upload/download to this node.
The "RunCommand" works fine too with output in the device agent console.

**Comment by Ylianst** [comment]
Your right. Thanks for reporting this. Just fixed it in v0.7.40. Update and let me know if it works.

**Comment by adsgreen** [comment]
Looks good.
